frontend/app/performance/lighthouse/constants.mjs

```diff
@@ -47,6 +47,8 @@ export const PERFORMANCE_APPS_DIRECTORY = path.resolve(
   "./frontend/app/performance/apps"
 )
 
+export const MULTIPAGE_APPS = ["./multipage/multipage_app.py"]
```

Comment:
nit: if we expect more multipage apps in the future, I am wondering whether we should name `./multipage/` differently, maybe `./multipage1/` or so to establish a naming pattern and make it clear that each multipage app goes into its own subdirectory. But it can also be done in the future when more are added.

Reply:
Good question! I'll have to write more up on this, but I don't expect the amount of Lighthouse apps to change too much over time. The reason is that Lighthouse is a good measure of initial page performance + accessibility, so just getting a wide but not necessarily deep coverage is desired here.

Other performance tools such as python-level metrics and Playwright interaction measurements, which are coming up next, are likely better ways of measuring more fine-grained parts.